jetstream/resource_jetstream_consumer.go

```diff
@@ -607,7 +608,7 @@ func resourceConsumerRead(d *schema.ResourceData, m any) error {
 
 	var bo []any
 	for _, d := range cons.Backoff() {
-		bo = append(bo, d.String())
+		bo = append(bo, d.Seconds())
```

Comment:
hmm, the `backoff` is a duration well the docs are not clear but I think should support like `1m` also so thats what the `String()` here was trying to do.  If there's old terraform code that had `1m` would this cause changes?

Reply:
As discussed, writing strings to Backoff didn't work before. The only valid values are ints. Updating the docs to reflect this.